core/model/src/main/java/org/eclipse/rdf4j/model/util/RDFContainers.java

```diff
@@ -0,0 +1,397 @@
+ * literals that are alternatives (typically for a single value of a property).
+ *
+ * So, in each of the above types, the container starts with a first resource node, via the rdf:_1 relation. Similarly,
+ * the next member is connected via the rdf:_2 relation and so on.
+ *
+ * For eg. Bag containing three literal values "A", "B", and "C" looks like this as an RDF Container:
+ *
+ * <pre>
+ *   _:n1 -rdf:type--> rdf:Bag
+ *     |
+ *     +---rdf:_1--> "A"
+ *     |
+ *     +---rdf:_2--> "B"
+ *     |
+ *     +---rdf:_3--> "C"
+ * </pre>
+ *
+ * 
+ * @see <a href="https://www.w3.org/TR/rdf-schema/#ch_container">RDF Schema 1.1 section on Collection vocabulary</a>.
+ */
+public class RDFContainers {
+
+	/**
+	 * Converts the supplied {@link Iterable} to an <a href="https://www.w3.org/TR/rdf-schema/#ch_container">RDF
+	 * Container</a>, using the supplied {@code head} resource as the starting resource of the RDF Containter. The
+	 * statements making up the new RDF Containter will be added to the supplied statement collection.
+	 *
+	 * @param containerType defines the type of RDF Container
+	 * @param values        an {@link Iterable} of objects (such as a Java {@link Collection} ), which will be converted
+	 *                      to an RDF Containter. May not be {@code null}. The method attempts to convert each value
+	 *                      that is not already an instance of {@link Value} to a {@link Literal}. This conversion will
+	 *                      fail with a {@link LiteralUtilException} if the value's object type is not supported. See
+	 *                      {@link Literals#createLiteralOrFail(ValueFactory, Object)} for an overview of supported
+	 *                      types.
+	 * @param container     a {@link Resource} which will be used as the head of the container, that is, the starting
+	 *                      point of the created RDF Container. May be {@code null}, in which case a new resource is
+	 *                      generated to represent the container head.
+	 * @param sink          a {@link Collection} of {@link Statement} objects (for example a {@link Model}) to which the
+	 *                      RDF Collection statements will be added. May not be {@code null}.
+	 * @param contexts      the context(s) in which to add the RDF Containter. This argument is an optional vararg and
+	 *                      can be left out.
+	 * @return the supplied sink {@link Collection} of {@link Statement}s, with the new Statements forming the RDF
+	 *         Collection added.
+	 * @throws LiteralUtilException if one of the supplied values can not be converted to a Literal.
+	 * @see <a href="https://www.w3.org/TR/rdf-schema/#ch_container">RDF Schema 1.1 section on Collection
+	 *      vocabulary</a>.
+	 */
+	public static <C extends Collection<Statement>> C asRDF(IRI containerType, Iterable<?> values, Resource container,
```

Comment:
Like @hmottestad also suggested above, I think renaming this to `toRDF` (and similarly, also rename `asValues` to `toValues`) is better. I realize you followed the naming we used in RDFCollections for this but I think we got this wrong there, and we might as well do it right in this one (we'll fix it in the RDFCollections later as well, but that's a separate issue).

Reply:
> Like @hmottestad also suggested above, I think renaming this to `toRDF` (and similarly, also rename `asValues` to `toValues`) is better. I realize you followed the naming we used in RDFCollections for this but I think we got this wrong there, and we might as well do it right in this one (we'll fix it in the RDFCollections later as well, but that's a separate issue).

@hmottestad @jeenbroekstra Unable to reply [here](https://github.com/eclipse/rdf4j/pull/2254#discussion_r429586815) because GitHub added the `outdated` tag to the conversation there. Yes, I can change the naming from `as*` to `to*`. However, I have to do this in `RDFContainers.java` only, right?